test/CMakeLists.txt

```diff
@@ -201,8 +201,8 @@ add_executable(
     test_ungqr.cc
     test_ungrq.cc
     test_ungtr.cc
+    test_unhr_col.cc    test_orhr_col.cc
```

Comment:
In a departure from other files, I put these 2 on the same line so it would be easy to see if both the `un` and `or` versions are done. Otherwise, the `or` version is alphabetized far above. Obviously most `un` routines are missing the `or` versions.

Reply:
When you say "done" do you mean when compiling, they will happen back to back so if there is an issue, we will catch it?

Moving away from the convention is a little ugly but I can understand your desire to do so, for this special case.